ui/src/AudioRecorder.js

```diff
@@ -34,12 +51,13 @@ export class AudioRecorder {
                 this.mediaRecorder.ondataavailable = (e) => {
                     const blob = new Blob([e.data], { type : 'audio/webm;' });
                     this.recordingUrl = window.URL.createObjectURL(blob);
-                    resolve(this.recordingUrl);
+                    const obj = [this.recordingUrl, blob];
```

Comment:
It might be clearer to return this as an object:
{
url: this.recordingUrl,
blob
}

Because then you can do:
const result = await stopRecording();
doSomething(result.url);
doSomething(result.blob);